source/main.cc

```diff
@@ -414,6 +413,37 @@ int main (int argc, char *argv[])
           return 2;
         }
 
+      // check whether ASPECT was called with --version or -v.
+      //
+      // for the same reason as above, ignore further arguments if we
+      // are using PETSc
+      if ((
+#ifdef ASPECT_USE_PETSC
+            argc >= 2
+#else
+            argc == 2
+#endif
+          )
+          &&
+          ((std::string (argv[1]) == "--version")
+           ||
+           (std::string (argv[1]) == "-v"))
+         )
+        {
+          // print usage info only on processor 0
+          if (Utilities::MPI::this_mpi_process(MPI_COMM_WORLD) == 0)
+            print_aspect_header(std::cout, true);
+
+          std::exit (0);
+        }
+
+
+
+      // regular operation: read an input file and run with it
```

Comment:
This comment confused me because I thought it explains this block. Maybe remove it?